.kitchen.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+provisioner:
+  name: terraform
+
+platforms:
+  - name: local
+
+verifier:
+  name: terraform
+  systems:
+    - name: system
+      backend: local
+
+suites:
+  - name: full
+    driver:
+      name: terraform
+      command_timeout: 1800
+      root_module_directory: test/fixtures/full
+      variables:
```

Comment:
Specifying these variables might be extremely surprising, since we normally specify all variables in a `terraform.tfvars` file. Can we specify this in the normal `terraform.tfvars` file?

Reply:
Will be dropped in the next release